File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task35_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static int peopleOnBus(int[][] stops) {
        int people = 0;
        for (int[] stop : stops) {
            people += stop[0] - stop[1];
        }
        return people;
    }
    public static void main(String[] args) {
        int[][] stops = {{10, 0}, {3, 5}, {5, 8}};
        System.out.println(peopleOnBus(stops));
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task35_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>

int peopleOnBus(int stops[][2], int size) {
    int people = 0;
    for (int i = 0; i < size; i++) {
        people += stops[i][0] - stops[i][1];
    }
    return people;
}

int main() {
    int stops[][2] = {{10, 0}, {3, 5}, {5, 8}};
    int size = sizeof(stops) / sizeof(stops[0]);
    printf("%d
", peopleOnBus(stops, size));
    return 0;
}